_posts/2024/03/2024-03-18-cage-recursion.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: post
+date: 2024-03-18
+title: "Recursion in cage"
+author: levBagryansky
+---
+
+The main difference between `cage` and `memory` is how the object is written.
+When dataizing, `memory.write` executes its argument as well. Whereas 
+`cage.write` just saves a reference to this object. As a result, the execution
+of the `cage` may turn out to be recursive. Unfortunately, this error is often not obvious.
+
+In this post we will tell you how we solved this problem.
+
+<!--more-->
+
+# A little about cage
+
+In [this](../../2023/08/2023-08-05-storing-objects-formed-differently-into-cage.md) post we 
```

Comment:
@maxonfjvipon how to refer to this post?

Reply:
@levBagryansky use the link to blog in the internet: https://news.eolang.org/

Reply:
@maxonfjvipon thanks